test/controllers/api/v2/packages_controller_test.rb

```diff
@@ -9,6 +9,7 @@ def models
       @host = hosts(:one)
       @simple_filter = katello_content_view_filters(:simple_filter)
       @one_package_rule = katello_content_view_package_filter_rules(:one_package_rule)
+      @org = ::Organization.find_by(name: "Empty Organization")
```

Comment:
we have a helper, `get_organization`, that will load an org from fixtures so I'd recommend using that. it loads the empty organization by default (no argument) otherwise it loads the fixture name specified!

Reply:
Thanks, I've pushed the changes now.